include/librealsense2/hpp/rs_sensor.hpp

```diff
@@ -23,13 +23,16 @@ namespace rs2
             error::handle(e);
             _category = rs2_get_notification_category(notification, &e);
             error::handle(e);
+            _serizlied_data = rs2_get_notification_serizlied_data(notification, &e);
+            error::handle(e);
         }
 
         notification()
             : _description(""),
```

Comment:
remove this constructor and put the initial values at the member declaration [C++ Core Guideline](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#c45-dont-define-a-default-constructor-that-only-initializes-data-members-use-in-class-member-initializers-instead)

Reply:
done